Jenkinsfile

```diff
@@ -1,5 +1,33 @@
 #!/usr/bin/env groovy
 // vim: set et sw=4 ts=4 :
+
+String distSubDir() {
```

Comment:
A bit of nit. Based on my understanding of the code this function returns the destination directory where the zip will be/is stored.
So maybe name this `distDestDir` ? Feels clearer to me.
Or maybe `distStoreDir` to account for the other use I saw, when the test stage pulls the zip from S3.

Reply:
Right, I called it that because it replaced the `subdir` variable that already existed. This is the `master/` part of `s3.amazonaws.com/cf-opensusefs2/scf/config/master/scf-xxxyyyzzz.zip`. (It's not used outside of S3.)

Reply:
I am ok with keeping as is, as said, just a nit. However please make the decision (keep/change) explicit here in the comment.

Reply:
Ah. I'm keeping it, then :)

Reply:
Ok. Now waiting for the jenkins to complete.